app/src/main/java/com/clockworkant/base/BaseActivityWithPresenter.kt

```diff
@@ -0,0 +1,26 @@
+package com.clockworkant.base
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+
```

Comment:
Class to extend to enable MVP on activities (screens).
When a user comes to the screen (onResume) it subscribes for updates
When a user leaves the screen (OnPause) it stops listening for updates.